README.md

```diff
@@ -17,18 +17,18 @@ Other terms used below:
 A minimalistic and stateless version of dusk-blockchain node where Consensus protocol in conjuction with Kadcast is able to join and participate `dusk-blockchain`/test-harness. Useful for testing and diagnostic incompatibility issues. This executable should be deprecated once dusk-blockchain is fully migrated.
 
 ## Example Testbed
-A multi-instance setup running 10 SBA instances provisioned with 10 eligible participants. The setup is configured to run for up to 1000 rounds. Useful for any kind of testing (issues, stress and performance testing).
+A multi-instance setup running 10 SA instances provisioned with 10 eligible participants. The setup is configured to run for up to 1000 rounds. Useful for any kind of testing (issues, stress and performance testing).
```

Comment:
What do you mean by SA instances? Maybe this should be clarified before

Reply:
Not sure how can I elaborate it. Feel free to read the code and suggest a better way.